src/componets/Comments.jsx

```diff
@@ -2,28 +2,34 @@ import { useState, useEffect } from "react";
 import Comment from "./CommentCard";
 
 export default function Comments({ singleArticle }) {
-  const [comments, setComments] = useState([]);
+  const [allComments, setAllComments] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
     setIsLoading(true);
     fetch(
```

Comment:
The reason this is not working as expected is because the default method that fetch will use is a GET request, so the request you are making isn't patching the database, its doing a GET request to article comments. If you want this to work you will have to look at how to change this to a PATCH request (you could look at how you have created the POST method for adding a comment as an example of how to use a fetch method that isn't GET)